ci-droid-gui/src/app/routes/form/form.component.ts

```diff
@@ -20,6 +21,8 @@ export class FormComponent implements OnInit {
   hide = false;
   fields: Field[];
   enablePreview = false;
+  success;
```

Comment:
what is the type ?

Reply:
It was removed by mistake Paul, I 'll add it.